src/main/java/lotto/game/LottoCount.java

```diff
@@ -0,0 +1,46 @@
+package lotto.game;
+
+import lotto.money.Money;
+
+import java.util.Objects;
+
+public class LottoCount {
+    public static final int ZERO = 0;
+    public static final int ONE_COUNT = 1;
+    private static final int LOTTO_PRICE = 1000;
+
+    private final int lottoCount;
+
+    public LottoCount(Money money) {
+        this.lottoCount = money.divideMoney(LOTTO_PRICE);
+    }
+
+    private LottoCount(int lottoCount) {
+        this.lottoCount = lottoCount;
+    }
```

Comment:
```suggestion
    public LottoCount(Money money) {
        this(money.divideMoney(LOTTO_PRICE));
    }

    private LottoCount(int lottoCount) {
        this.lottoCount = lottoCount;
    }
```

인스턴스 변수의 초기화는 하나의 생성자에서 해주는 것이 좋은 이유에 대해 고민해보아요!